src/creator.coffee

```diff
@@ -234,6 +239,13 @@ Hangman.controller 'HangmanCreatorCtrl', ['$timeout', '$scope', '$sanitize', 'Re
 		,
 		400
 
+	$scope.openQuestionBankDialog = ->
+		$scope.questionBankDialog = true
+		$scope.enableQuestionBank = false
```

Comment:
What's the purpose of setting `enableQuestionBank` to false here? The way you have this setup, even if I previously set `enableQuestionBank` to `true`, opening the dialog flips the value back to `false`.